packages/scrum-server/src/services/socket/ServerSocketState.js

```diff
@@ -46,7 +46,7 @@ const ServerSocketState = (initialState = []) => {
     logger.debug(`setRoomStoryVote on room {${room.id}} for user {${userId}} with value {${vote}}`);
     const _room = rooms.get(room.id);
     _room.storyVotes.set(userId, vote);
-    logger.silly(`setRoomStoryVote on room {${room.id}} for user {${userId}} has {${JSON.stringify(_room.storyVotes)}}`);
+    logger.silly(`setRoomStoryVote on room {${room.id}} for user {${userId}} has {${JSON.stringify(Array.from(_room.storyVotes.entries()))}}`);
```

Comment:
I think at some point we may need to improve this. I think besides this is silly level cpu is still consumed due to the transformation of the data before the log is even called. We will need a strategy anyway but all good here

Reply:
socket state  is not meant to be in memory ....  i would imagine this needs to be ported to mongodb, right ?

Reply:
right